Profiles/StructureDefinition-onco-core-Procedure-Operation.xml

```diff
@@ -79,11 +79,15 @@
     <element id="Procedure.code.coding:OPS">
       <path value="Procedure.code.coding" />
       <sliceName value="OPS" />
+      <type>
+        <code value="Coding" />
+        <profile value="http://fhir.de/StructureDefinition/CodingOPS" />
+      </type>
     </element>
     <element id="Procedure.code.coding:OPS.system">
```

Comment:
I'm not so familiar with profiling. Do we have to repeat the fixed system here if it is already defined in the CodingOPS profile?